pyaccel/tracking.py

```diff
@@ -810,6 +819,13 @@ def find_m44(accelerator, indices='m44', energy_offset=0.0, fixed_point=None):
 
 # ------ Auxiliary methods -------
 
+
+def _set_random_seed():
+    # seed = _np.random.randint(0, 2**32-1)  # introduces correlation with parallel tracking!
+    seed = _random.randint(0, 2**32-1)  # c++ seed is a 32-bit unsigned int
+    _trackcpp.set_random_seed(seed)
```

Comment:
The problem with this solution is that we loose control over the repeatability of the random numbers we create.

Reply:
yes. this needs changing for the final PR. I will add the solution we discussed offline in trackcpp.

Reply:
done

Reply:
I think this problem will remain, even with the approach we discussed offline. Maybe we have to talk again to brainstorm about it.

Reply:
we decided to merge this one as is and address this issue of repeatability in future PRs.